app/src/main/java/org/cru/godtools/adapter/TextViewAdapter.kt

```diff
@@ -0,0 +1,37 @@
+package org.cru.godtools.adapter
+
+import android.net.Uri
+import android.text.SpannableString
+import android.text.Spanned
+import android.text.method.LinkMovementMethod
+import android.text.style.URLSpan
+import android.util.Patterns
+import android.view.View
+import android.widget.TextView
+import androidx.databinding.BindingAdapter
+import org.cru.godtools.base.tool.analytics.model.ExitLinkActionEvent
+import org.greenrobot.eventbus.EventBus
+
+@BindingAdapter("webLinkText")
+fun TextView.setTextViewWebLinksForAnalytics(
+    changedText: CharSequence
+) {
+    val spannable = SpannableString(changedText)
+    val matcher = Patterns.WEB_URL.matcher(spannable)
+    while (matcher.find()) {
+        var url = spannable.toString().substring(matcher.start(), matcher.end())
+        if (!url.startsWith("http")) {
+            url = "http://$url"
+        }
+        val urlSpan = object : URLSpan(url) {
+            override fun onClick(widget: View) {
+                val uri = Uri.parse(url)
+                EventBus.getDefault().post(ExitLinkActionEvent(uri))
+                super.onClick(widget)
+            }
+        }
+        spannable.setSpan(urlSpan, matcher.start(), matcher.end(), Spanned.SPAN_EXCLUSIVE_EXCLUSIVE)
+    }
+    text = spannable
+    movementMethod = LinkMovementMethod.getInstance() // Make link clickable
+}
```

Comment:
what are your thoughts on this logic for the binding adapter: https://github.com/CruGlobal/android-gto-support/pull/436/files

Reply:
The only issue I see is what I ran into a link that doesn't start with HTTP will not open when you click it.  Other than that it looks great.

Reply:
hmm, I would push back that links that don't start with http:// should be updated to be proper URLs since we do control that content.

Reply:
although if that is a blocker I believe there is an option for LinkifyCompat to automatically add a uri scheme.

Reply:
I'll research shortly

Reply:
<img width="365" alt="Android_Emulator_-_Pixel_3a_API_29_5554" src="https://user-images.githubusercontent.com/970048/75998432-d9e29300-5ece-11ea-80fa-03561f281602.png">
These are the two that gave me the issue.

Reply:
I just tested on those links and they worked fine with the LinkifyCompat logic